internal/cli/push.go

```diff
@@ -155,7 +164,7 @@ var CmdPush = &cobra.Command{
 		}
 		params.Services = services
 
-		err = client.Push(cmd.Context(), args[0], path, params)
+		err = client.Push(cmd.Context(), params)
```

Comment:
Can we report the builder image in the this output?

```

🚢 About to push an application with given name and sources into the specified organization
Name: sample
Sources: assets/sample-app/
Organization: workspace
```

Reply:
Done.